kmod/src/server.c

```diff
@@ -627,6 +627,9 @@ static int reclaim_log_trees(struct super_block *sb, u64 rid)
 					&lt.meta_avail) ?:
 	      alloc_move_empty(sb, &super->data_alloc, &lt.data_avail) ?:
 	      alloc_move_empty(sb, &super->data_alloc, &lt.data_freed);
+	if (lt.oino_bloom_ref.blkno && !ret)
+		ret = scoutfs_free_meta(sb, &server->alloc, &server->wri,
+					le64_to_cpu(lt.oino_bloom_ref.blkno));
```

Comment:
Looks right, but we'd want to zero that ref blkno (and maybe the whole ref including the seq?) to indicate that the block has been freed.  The log_trees item will still sit around for a while so that the item log tree and item bloom can be used by readers.

Reply:
zero the ref, got it.